build.assets/build-fido2-macos.sh

```diff
@@ -11,13 +11,14 @@
 # Written mainly for macOS builders.
 set -eu
 
+readonly MACOS_VERSION_MIN=10.12
```

Comment:
Go 1.17 requires [macOS 10.13](https://go.dev/doc/go1.17#darwin) or later, so I think you can safely set this to 10.13.

Reply:
Done.